libs/application/templates/new-primary-school/src/graphql/queries.ts

```diff
@@ -37,6 +41,10 @@ export const friggSchoolsByMunicipalityQuery = gql`
         email
         phone
         website
+        address {
+          municipality
+          postalCode
```

Comment:
`postalCode` is misspelled (should be `postCode`)
```suggestion
          postCode
```

Reply:
Not being used, removed